pubber/src/main/java/daq/pubber/Pubber.java

```diff
@@ -575,12 +579,15 @@ private void maybeRestartSystem() {
     if (systemConfig == null) {
       return;
     }
+    info("maybeRestartSystem " + deviceState.system.mode + " " + systemConfig.mode
```

Comment:
I think this is likely intermediate debug info, so should be removed or downgraded to debug?

Reply:
Agree, debug